cmake/PybindWrap.cmake

```diff
@@ -35,9 +42,16 @@ function(pybind_wrap
   else(USE_BOOST)
     set(_WRAP_BOOST_ARG "")
   endif(USE_BOOST)
-  
+
+  if (UNIX)
+    set(GTWRAP_PATH_SEPARATOR ":")
+  else()
+    set(GTWRAP_PATH_SEPARATOR ";")
+  endif()
+
   add_custom_command(OUTPUT ${generated_cpp}
-                     COMMAND ${PYTHON_EXECUTABLE}
+          COMMAND ${CMAKE_COMMAND} -E env "PYTHONPATH=${GTWRAP_PACKAGE_DIR}${GTWRAP_PATH_SEPARATOR}$ENV{PYTHONPATH}"
```

Comment:
Does this set the environment path across sessions? If yes, then awesome!

Reply:
Yes, this sets the PYTHONPATH at compile time :)

Reply:
Pardon me I guess my question was vague. I was wondering if I install wrap in one terminal (which sets the PYTHONPATH) and then try to wrap a project in another terminal, will it carry over to the other terminal? Actually let me try it out.

Reply:
Actually wait, wrong question. This is in the PybindWrap.cmake file so it is invoked during wrapping. Does this mean the user needs to specify `GTWRAP_DIR` in their project? What would that be? The location where the wrapper scripts are installed (e.g. in my case `/usr/local/bin/`)?

Reply:
This is to enable GTSAM builds without a global install of `wrap`. When not specified it defaults to `<source>/wrap`, which is the old behavior, but when there is `GTWRAP_DIR`, i.e., user invokes `find_package(Gtwrap)` to find a global install, it uses `GTWRAP_DIR`.

Reply:
I think to truly enable `gtwrap` to be used anywhere after a global install the config cmake file needs to be rewritten and the location of the python files needs to be tweaked. Most likely the package should be installed to `<prefix>/lib/share/wrap` and in the config cmake file we try searching this path. I will do this in a later PR.

Reply:
This should be part of this PR otherwise the wrapping will be broken. The python files also need to be installed so that they can be found by the specified PYTHONPATH which isn't currently happening.